docs/guide/docs/channels/vonage.md

```diff
@@ -0,0 +1,62 @@
+---
+id: vonage
+title: Smooch (Sunshine Conversations)
```

Comment:
Should be changed to 'Vonage'. And also, please note that all channels documentation will be updated with [this PR](https://github.com/botpress/botpress/pull/5220/files?file-filters%5B%5D=.md).